packages/sources/durin/schemas/env.json

```diff
@@ -0,0 +1,25 @@
+{
+  "$id": "https://external-adapters.chainlinklabs.com/schemas/durin.json",
+  "title": "@chainlink/durin env var schema",
+  "required": [],
+  "type": "object",
+  "properties": {
+    "RPC_URL": {
+      "required": true,
```

Comment:
Are these required fields used anywhere else? Long-term I would like to see a single source of truth, either these `required` attributes or the `required` array, but not both

Reply:
I'm actually not sure about this.  @justinkaseman do you know the answer?